pkgs/by-name/ge/gerrit-queue/package.nix

```diff
@@ -0,0 +1,25 @@
+{ buildGoModule
+, lib
+, fetchFromGitHub
+}:
+
+buildGoModule {
+  pname = "gerrit-queue";
+  version = "unstable-2023-10-20";
```

Comment:
If it's good enough to be packaged, how about cutting a release as well?

Reply:
There's some more things I'd like to address before cutting a release, like cleaning up the whole log/web interface story. I'd put it not in the critical path for the package recipe.

Reply:
I applied some small fixes in https://github.com/flokli/gerrit-queue/pull/16, and tagged a v0.0.1. PTAL.